### DISCUSSION TITLE
EC2MetadataUtils.IamSecurityCredentials class is missing in secretsmanager v2.17.233 (we need to fix on this version due to certain other dependencies).

### DISCUSSION THREAD

**Discussion by damienborowski** [aws/aws-sdk-java#2816] [Q&A] [answered]
Any idea what it's replacement class is, or how in this version are we meant to get the tokens, access keys etc ? I see it's 'back' in 2.9.9 but we can't use this unfortunately.

Would be happy to explain further/more detailed if needs be

**Comment by yasminetalby** [comment, marked as answer]
Hello @damienborowski ,

Thank you very much for your message. Please refer the [ issue submission ](https://github.com/aws/aws-sdk-java/issues/2817) to avoid duplicate threads.

Best,

Yasmine
